FindOpenGLES2.cmake

```
@@ -0,0 +1,38 @@
+# - Try to find OpenGLES2
+# Once done this will define
+#
+#  OPENGLES2_FOUND        - system has OpenGLES2
+#  OPENGLES2_INCLUDE_DIR  - the GLES2 include directory
+#  OPENGLES2_LIBRARIES    - Link these to use OpenGLES2
+
+find_package(PkgConfig)
+if(PKG_CONFIG_FOUND AND NOT EMBEDDED_FOUND)
```

This module doesn't define `EMBEDDED_FOUND`.
```suggestion
if(PKG_CONFIG_FOUND)
```

---

Bad c&p, has taken them from pvr.vdr.vnsi.